auth/token-managers/iam-token-manager.ts

```diff
@@ -35,12 +35,14 @@ function onlyOne(a: any, b: any): boolean {
 }
 
 const CLIENT_ID_SECRET_WARNING = 'Warning: Client ID and Secret must BOTH be given, or the header will not be included.';
+const SCOPE = 'scope';
 
 /** Configuration options for IAM token retrieval. */
 interface Options extends JwtTokenManagerOptions {
   apikey: string;
   clientId?: string;
   clientSecret?: string;
+  string?: string;
```

Comment:
I think this is meant to be `scope`?
```suggestion
  scope?: string;
```

Reply:
oh wow, yep completely missed this..